src/macros.rs

```diff
@@ -417,7 +445,9 @@ pub(crate) fn rewrite_macro_def(
     let mut parser = MacroParser::new(ts.trees());
     let parsed_def = match parser.parse() {
         Some(def) => def,
-        None => return snippet,
+        None => {
+            return snippet;
+        }
```

Comment:
Did this need to change? I don't think I would have expected to see a diff here.